optima/results.py

```diff
@@ -552,7 +557,47 @@ def export(self, filename=None, folder=None, bypop=True, sep=',', ind=None, key=
             return fullpath
         else:
             return outputstr
-        
+
+    def comparebudgets(self, sep='\",\"', sigfigs=3):
+        """ Make a separate sheet in the workbook with a comparison of budget and coverage for an optimization """
+        outputstr = sep.join(['Programs', 'Baseline budget', '% baseline budget', 'Optimized budget',
+                              '% optimized budget', '% budget change',
+                              'Baseline coverage', 'Optimized coverage', '% coverage change'])  # Create headers
+        total_baseline_budget = sum(self.budgets['Baseline'].values())
+        total_optimized_budget = sum(self.budgets['Optimal'].values())
+        for prog, baseline_budget in self.budgets['Baseline'].items():
+            outputstr += '\n'
+
+            optimized_budget = self.budgets['Optimal'][prog]
+            baseline_cov = self.coverages['Baseline'][prog]
+            if checktype(baseline_cov, 'arraylike'):
+                baseline_cov = baseline_cov[0]  # Only pull out the first element if it's an array/list
+            if baseline_cov is None: baseline_cov = 0  # Just reset
+            optimized_cov = self.coverages['Optimal'][prog]
+            if checktype(optimized_cov, 'arraylike'):
+                optimized_cov = optimized_cov[0]  # Only pull out the first element if it's an array/list
+            if optimized_cov is None: optimized_cov = 0  # Just reset
+
+            budget_change = 0
+            cov_change = 0
+            if baseline_budget <= 0.1 < optimized_budget:
+                budget_change = 1.0
+            elif baseline_budget > 0.1:
+                budget_change = (optimized_budget - baseline_budget)/baseline_budget
+            if baseline_cov <= 0.1 < optimized_cov:
+                cov_change = 1.0
+            elif baseline_cov > 0.1:
+                cov_change = (optimized_cov - baseline_cov) / baseline_cov
+            outputstr += prog + sep + str(baseline_budget) + sep + \
+                         '%fperc' % (baseline_budget/total_baseline_budget) + sep + str(optimized_budget) + sep + \
```

Comment:
See above about comment using %% instead of perc

Reply:
perc was the label for indicating a cell was to be formatted as percent in excel. Updated to more unique label "!PERC"